ckan/controllers/api.py

```diff
@@ -287,7 +288,7 @@ def convert_to_dict(user):
             return out
 
         query = query.limit(limit)
-        out = map(convert_to_dict, query.all())
+        out = list(map(convert_to_dict, query.all()))
```

Comment:
out = [convert_to_dict(q) for q in query.all()]  # list comprehension

https://docs.python.org/3.0/whatsnew/3.0.html#views-and-iterators-instead-of-lists
> map() and filter() return iterators. If you really need a list, a quick fix is e.g. list(map(...)), but a better fix is often to use a list comprehension (especially when the original code uses lambda), or rewriting the code so it doesn’t need a list at all. Particularly tricky is map() invoked for the side effects of the function; the correct transformation is to use a regular for loop (since creating a list would just be wasteful).